Beta_Integrations/OpenLDAP/OpenLDAP.yml

```diff
@@ -0,0 +1,156 @@
+category: Authentication
+commonfields:
+  id: OpenLDAP
+  version: -1
+configuration:
+  - display: Fetch incidents
+    name: isFetch
+    required: false
+    type: 8
+  - display: Incident type
+    name: incidentType
+    required: false
+    type: 13
+  - display: Server IP or Host Name (e.g. 192.168.0.1)
+    name: host
+    required: true
+    type: 0
+  - display: Port. If not specified, default the port is 389, or 636 for LDAPS.
+    name: port
+    required: false
+    type: 0
+  - display: Credentials
```

Comment:
lets find better display name

Reply:
@Itay4 you mean instead of `Credentials`?